lib/ssl/src/tls_server_session_ticket.erl

```diff
@@ -385,10 +385,10 @@ stateless_living_ticket(0, _, _, _, _) ->
 stateless_living_ticket(ObfAge, TicketAgeAdd, Lifetime, Timestamp, Window) ->
     ReportedAge = ObfAge - TicketAgeAdd,
     RealAge = erlang:system_time(second) - Timestamp,
```

Comment:
why you replaced `system_time` with `monotonic_time` in tls_client_ticket_store but left it in `tls_server_session_ticket`?

Reply:
I kept `system_time` in `tls_server_session_ticket` because this time stamp is stored in the encrypted stateless ticket and it could potentially be generated in one server instance and used in a different server instance. This doesn't work currently, since the encryption seed would be different between instances, but I have another issue that would make it possible (#5962). It doesn't make sense to compare monotonic times between difference BEAM instances. Hence, I prefer to keep `system_time` with those things in mind.